### PR TITLE
Popup details

### PR DESCRIPTION
**Pull request by SimbaWANGU**
In this PR, the following were done

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,56 @@
 </head>
 
 <body>
+    <div class="popup-window-container hide">
+        <div class="popup-window">
+          <div class="project-heading">
+            <h2 class="popup_title"></h2>
```
Comment (VICTOR-CHUKWU) [inline index.html:17]: - [x] Nice work 👍 , Kindly Fix your heading in the desktop view
    how your design looks 👇 
      
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @S-imba 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimbaWANGU** [comment]
I have updated the project with the requested changes.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @S-imba , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimbaWANGU** [comment]
Thank you @VICTOR-CHUKWU

**Pull request merged**